### PR TITLE
HBASE-22395 Document RC voting guidelines in ref guide

### PR DESCRIPTION
**Pull request by taklwu**

### CODE REVIEW

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 66 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | @author | 0 | The patch does not contain any @author tags. |
||| _ master Compile Tests _ |
| +1 | mvninstall | 262 | master passed |
| 0 | refguide | 444 | branch has no errors when building the reference guide. See footer for rendered docs, which you should manually inspect. |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 242 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| 0 | refguide | 435 | patch has no errors when building the reference guide. See footer for rendered docs, which you should manually inspect. |
||| _ Other Tests _ |
| +1 | asflicense | 14 | The patch does not generate ASF License warnings. |
| | | 1528 | |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=17.05.0-ce Server=17.05.0-ce base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-297/1/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/297 |
| Optional Tests |  dupname  asflicense  refguide  |
| uname | Linux e9c3c6c89995 4.4.0-138-generic #164-Ubuntu SMP Tue Oct 2 17:16:02 UTC 2018 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /testptch/patchprocess/precommit/personality/provided.sh |
| git revision | master / 03436e68bc |
| maven | version: Apache Maven 3.5.4 (1edded0938998edf8bf061f1ceb3cfdeccf443fe; 2018-06-17T18:33:14Z) |
| refguide | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-297/1/artifact/out/branch-site/book.html |
| refguide | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-297/1/artifact/out/patch-site/book.html |
| Max. process+thread count | 96 (vs. ulimit of 10000) |
| modules | C: . U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-297/1/console |
| Powered by | Apache Yetus 0.9.0 http://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 159 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | @author | 0 | The patch does not contain any @author tags. |
||| _ master Compile Tests _ |
| +1 | mvninstall | 268 | master passed |
| 0 | refguide | 466 | branch has no errors when building the reference guide. See footer for rendered docs, which you should manually inspect. |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 251 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| 0 | refguide | 449 | patch has no errors when building the reference guide. See footer for rendered docs, which you should manually inspect. |
||| _ Other Tests _ |
| +1 | asflicense | 17 | The patch does not generate ASF License warnings. |
| | | 1663 | |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=17.05.0-ce Server=17.05.0-ce base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-297/2/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/297 |
| Optional Tests |  dupname  asflicense  refguide  |
| uname | Linux 4dc115f7da10 4.4.0-139-generic #165-Ubuntu SMP Wed Oct 24 10:58:50 UTC 2018 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /testptch/patchprocess/precommit/personality/provided.sh |
| git revision | master / ada772a1d3 |
| maven | version: Apache Maven 3.5.4 (1edded0938998edf8bf061f1ceb3cfdeccf443fe; 2018-06-17T18:33:14Z) |
| refguide | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-297/2/artifact/out/branch-site/book.html |
| refguide | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-297/2/artifact/out/patch-site/book.html |
| Max. process+thread count | 78 (vs. ulimit of 10000) |
| modules | C: . U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-297/2/console |
| Powered by | Apache Yetus 0.9.0 http://yetus.apache.org |


This message was automatically generated.

File: src/main/asciidoc/_chapters/developer.adoc [hunk 1]
```diff
@@ -876,7 +876,18 @@ Regards the latter, run `mvn apache-rat:check` to verify all files are suitably
...
+To help with the release verification, please follow the guideline below and vote based on the your verification.
+
+=== Baseline Verifications for Voting Release Candidates
+
+Although contributors have their own checklist for verifications, the following items are usually used for voting on release candidates.
+
+* CHANGES.md if any
+* RELEASENOTES.md (release notes) if any
+* Generated API compatibility report
+** For what should be compatible please refer the [versioning guideline](https://hbase.apache.org/book.html#hbase.versioning), especially for items with marked as high severity
```
Comment (busbey) [inline src/main/asciidoc/_chapters/developer.adoc]: this didn't render correctly as a link in the qabot build of the ref guide.

File: src/main/asciidoc/_chapters/developer.adoc [hunk 2]
```diff
@@ -899,6 +910,15 @@ Usage: hbase-vote.sh -s | --source <url> [-k | --key <signature>] [-f | --keys-f
                                 http://www.apache.org/dist/hbase/KEYS
   -o | --output-dir '</path>'   directory which has the stdout and stderr of each verification target
 ----
+** If you see any unit test failures, please call out the solo test result and whether it's part of flaky (nightly) tests dashboard. Link to the dashboard is e.g. https://builds.apache.org/view/H-L/view/HBase/job/HBase-Find-Flaky-Tests/job/<branch-name>/lastSuccessfulBuild/artifact/dashboard.html
```
Comment (busbey) [inline src/main/asciidoc/_chapters/developer.adoc]: this rendered as a link to an incorrect URL. Can you make it an example of a working link with branch-2.2?

File: src/main/asciidoc/_chapters/developer.adoc (hunk 1 above)
Comment (taklwu) [reply src/main/asciidoc/_chapters/developer.adoc]: fixed, thanks for calling out

File: src/main/asciidoc/_chapters/developer.adoc (hunk 2 above)
Comment (taklwu) [reply src/main/asciidoc/_chapters/developer.adoc]: fixed, I changed to master branch with a note one should change the test branch accordingly such we don't have to change this ref guide often.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 80 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | @author | 0 | The patch does not contain any @author tags. |
||| _ master Compile Tests _ |
| +1 | mvninstall | 262 | master passed |
| 0 | refguide | 482 | branch has no errors when building the reference guide. See footer for rendered docs, which you should manually inspect. |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 238 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| 0 | refguide | 430 | patch has no errors when building the reference guide. See footer for rendered docs, which you should manually inspect. |
||| _ Other Tests _ |
| +1 | asflicense | 14 | The patch does not generate ASF License warnings. |
| | | 1562 | |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=17.05.0-ce Server=17.05.0-ce base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-297/3/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/297 |
| Optional Tests |  dupname  asflicense  refguide  |
| uname | Linux 01229e4429d5 4.4.0-138-generic #164-Ubuntu SMP Tue Oct 2 17:16:02 UTC 2018 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /testptch/patchprocess/precommit/personality/provided.sh |
| git revision | master / ada772a1d3 |
| maven | version: Apache Maven 3.5.4 (1edded0938998edf8bf061f1ceb3cfdeccf443fe; 2018-06-17T18:33:14Z) |
| refguide | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-297/3/artifact/out/branch-site/book.html |
| refguide | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-297/3/artifact/out/patch-site/book.html |
| Max. process+thread count | 76 (vs. ulimit of 10000) |
| modules | C: . U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-297/3/console |
| Powered by | Apache Yetus 0.9.0 http://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 38 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | @author | 0 | The patch does not contain any @author tags. |
||| _ master Compile Tests _ |
| +1 | mvninstall | 259 | master passed |
| 0 | refguide | 743 | branch has no errors when building the reference guide. See footer for rendered docs, which you should manually inspect. |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 250 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| 0 | refguide | 485 | patch has no errors when building the reference guide. See footer for rendered docs, which you should manually inspect. |
||| _ Other Tests _ |
| +1 | asflicense | 15 | The patch does not generate ASF License warnings. |
| | | 1859 | |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=17.05.0-ce Server=17.05.0-ce base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-297/4/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/297 |
| Optional Tests |  dupname  asflicense  refguide  |
| uname | Linux caa3c373a2f7 4.4.0-143-generic #169~14.04.2-Ubuntu SMP Wed Feb 13 15:00:41 UTC 2019 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /testptch/patchprocess/precommit/personality/provided.sh |
| git revision | master / ada772a1d3 |
| maven | version: Apache Maven 3.5.4 (1edded0938998edf8bf061f1ceb3cfdeccf443fe; 2018-06-17T18:33:14Z) |
| refguide | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-297/4/artifact/out/branch-site/book.html |
| refguide | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-297/4/artifact/out/patch-site/book.html |
| Max. process+thread count | 86 (vs. ulimit of 10000) |
| modules | C: . U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-297/4/console |
| Powered by | Apache Yetus 0.9.0 http://yetus.apache.org |


This message was automatically generated.

**Comment by taklwu** [comment]
I found an extra `the` and removed it as the last commit.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 42 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | @author | 0 | The patch does not contain any @author tags. |
||| _ master Compile Tests _ |
| +1 | mvninstall | 283 | master passed |
| 0 | refguide | 466 | branch has no errors when building the reference guide. See footer for rendered docs, which you should manually inspect. |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 234 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| 0 | refguide | 463 | patch has no errors when building the reference guide. See footer for rendered docs, which you should manually inspect. |
||| _ Other Tests _ |
| +1 | asflicense | 15 | The patch does not generate ASF License warnings. |
| | | 1569 | |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=17.05.0-ce Server=17.05.0-ce base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-297/5/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/297 |
| Optional Tests |  dupname  asflicense  refguide  |
| uname | Linux fd6d0f1e519d 4.4.0-139-generic #165-Ubuntu SMP Wed Oct 24 10:58:50 UTC 2018 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /testptch/patchprocess/precommit/personality/provided.sh |
| git revision | master / 934d469fd5 |
| maven | version: Apache Maven 3.5.4 (1edded0938998edf8bf061f1ceb3cfdeccf443fe; 2018-06-17T18:33:14Z) |
| refguide | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-297/5/artifact/out/branch-site/book.html |
| refguide | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-297/5/artifact/out/patch-site/book.html |
| Max. process+thread count | 76 (vs. ulimit of 10000) |
| modules | C: . U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-297/5/console |
| Powered by | Apache Yetus 0.9.0 http://yetus.apache.org |


This message was automatically generated.

**Pull request closed without merging**